notebooks/model_monitoring/solutions/model_monitoring_vertex.ipynb

```diff
@@ -0,0 +1,1396 @@
+{
```

Comment:
<ul><li>What is happening in this cell is very opaque. I think the code here needs to be broken down into smaller chunks and explained in markdown cells. I also suggest to completely get rid of all the helper functions and use the API directly, explaining what each call does and why.</li></ul><p> </p><ul><li>This cell failed for me, so I was not able to fully test the notebook to the end.</li></ul>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/asl-ml-immersion/pull/209/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='22'/>